site/content/docs/5.2/getting-started/vite.md

```diff
@@ -0,0 +1,197 @@
+│   |   └── styles.scss
+|   └── index.html
+├── package-lock.json
+├── package.json
+└── vite.config.js
+```
+
+At this point, everything is in the right place, but Vite won't work because we haven't filled in our `vite.config.js` yet.
+
+## Configure Vite
+
+With dependencies installed and our project folder ready for us to start coding, we can now configure Vite and run our project locally.
+
+1. **Open `vite.config.js` in your editor.** Since it's blank, we'll need to add some boilerplate config to it so we can start our server. This part of the config tells Vite were to look for our project's JavaScript and how the development server should behave (pulling from the `src` folder with hot reload).
+
+   <!-- eslint-skip -->
+   ```js
+   const path = require('path')
+
+   export default {
+     root: path.resolve(__dirname, 'src'),
+     server: {
+       port: 8080,
+       hot: true
+     }
+   }
+   ```
+
+2. **Next we fill in `src/index.html`.** This is the HTML page Vite will load in the browser to utilize the bundled CSS and JS we'll add to it in later steps.
+
+   ```html
+   <!doctype html>
+   <html lang="en">
+     <head>
+       <meta charset="utf-8">
+       <meta name="viewport" content="width=device-width, initial-scale=1">
+       <title>Bootstrap w/ Vite</title>
+     </head>
+     <body>
+       <div class="container py-4 px-3 mx-auto">
+         <h1>Hello, Bootstrap and Vite!</h1>
+         <button class="btn btn-primary">Primary button</button>
+       </div>
+       <script type="module" src="./js/main.js"></script>
+     </body>
+   </html>
+   ```
+
+   We're including a little bit of Bootstrap styling here with the `div class="container"` and `<button>` so that we see when Bootstrap's CSS is loaded by Vite.
+
+3. **Now we need an npm script to run Vite.** Open `package.json` and add the `start` script shown below (you should already have the test script). We'll use this script to start our local Vite dev server.
+
+   ```json
+   {
+     // ...
+     "scripts": {
+       "start": "vite",
+       "test": "echo \"Error: no test specified\" && exit 1"
+     },
+     // ...
+   }
+   ```
+
+4. **And finally, we can start Vite.** From the `my-project` folder in your terminal, run that newly added npm script:
+
+   ```sh
+   npm start
+   ```
+
+   <img class="img-fluid" src="/docs/{{< param docs_version >}}/assets/img/guides/vite-dev-server.png" alt="Vite dev server running">
+
+In the next and final section to this guide, we’ll import all of Bootstrap’s CSS and JavaScript.
+
+## Import Bootstrap
+
+1. **Set up Bootstrap Sass import in `vite.config.js`.** Your configuration file is now complete and should match the snippet below. The only new part here is the `resolve` section.
+
+   <!-- eslint-skip -->
+   ```js
+   const path = require('path')
+
+   export default {
+     root: path.resolve(__dirname, 'src'),
+     resolve: {
+       alias: {
+         '~bootstrap': path.resolve(__dirname, 'node_modules/bootstrap'),
```

Comment:
Not sure that's the best way to do it

Reply:
I've seen this mentioned in other guides as I've been rewriting Parcel and Webpack. I think it's good to go unless Vite has some other built-in aliasing.